.github/container/Dockerfile.maxtext

```diff
@@ -2,7 +2,7 @@
 
 ARG BASE_IMAGE=ghcr.io/nvidia/jax-mealkit:jax
 ARG URLREF_MAXTEXT=https://github.com/google/maxtext.git#main
-ARG URLREF_TFTEXT=https://github.com/tensorflow/text.git#v2.13.0
+ARG URLREF_TFTEXT=https://github.com/tensorflow/text.git#master
```

Comment:
Is this tested? It does not seem to be related to the bump to Ubuntu 24.04.

Reply:
Unfortunately, related. Google merge changes to support wheel build for python3.12 via this [PR](https://github.com/tensorflow/text/pull/1323). For now we have to use master  because the last release is 2.18.0 published 1 month ago and the fixing PR was merged 2 weeks ago. Waiting 2.18+ release to ping it here.

Reply:
Note, this make use track master TF-Text instead of a pinned version.
We could pin a commit as an alternative.

Reply:
> Note, this make use track master TF-Text instead of a pinned version. We could pin a commit as an alternative.

Our current 'standard' approach is to tracker the master in the Dockerfile, and then use `manifest.yaml` and the [`EXTRA_BUILD_ARGS`](https://github.com/NVIDIA/JAX-Toolbox/blob/b0e67537bca3955520f5503a0d869178eaf5d6ae/.github/workflows/_ci.yaml#L158) mechanism for CI reproducibility.

Reply:
@DwarKapex could you please add the plumbing for lingvo/tftext version tracking following the example of https://github.com/NVIDIA/JAX-Toolbox/blob/b0e67537bca3955520f5503a0d869178eaf5d6ae/.github/workflows/_ci.yaml#L158 ?